Observatory-Control-System/EAS/DIMM/ops.rst

```diff
@@ -106,8 +106,24 @@ and within the OpenTPL:
         AUTH OK 0 0
         1 set ameba.mode=1
         
+It is common to see ``EVENT WARN`` messages regarding the state of ameba. This does not necessarily indicate an issue.
 
-Verify the DIMM CSC is enabled and data is appearing in the EFD.
+The status of ameba can be checked with with following command to *tt-master*. This can be useful to check if the DIMM will operate tonight, or if there were concerning warning messages when starting ameba.
```

Comment:
Consider adding this section of "Checking Status of ameba" in a differentiated box, for instance, using the directive admonition

```
.. admonition:: Status of ameba

        The status of ameba can be checked with with following command to *tt-master*. This can be useful to 
        check if the DIMM will operate tonight, or if there were concerning warning messages when starting 
        ameba.

        .. code-block:: bash

                AUTH OK 00
                1 get ameba.mode

        Which will reply something like the following with 1 or 0 to indicate on or off.

        .. code-block:: bash

                1 COMMAND OK
                1 DATA INLINE AMEBA.MODE=1
                1 COMMAND COMPLETE      

```

would look like 
![Screen Shot 2024-06-04 at 11 45 36](https://github.com/lsst-ts/observatory-ops-docs/assets/87160242/9ecbd8ac-09d4-42e0-b9d0-31038e4f6f01)